### PR TITLE
Added Projects Section

### PR DESCRIPTION
**Pull request by vikytech**
Added ability to render **Projects** section

### CODE REVIEW

**Comment by vikytech** [comment]
@mudassir0909  / Any other contributor
Can we please have a look into this PR

**Comment by mudassir0909** [comment]
@vikytech hey, thanks for the pull request but is the `projects` field part of the official JSON resume schema? I couldn't find it [here](https://jsonresume.org/schema/)

**Comment by mudassir0909** [comment]
Please ignore my previous comment, looks like it is part of the official schema as [per Github](https://github.com/jsonresume/resume-schema/blob/v1.0.0/schema.json)

Few comments:
* Please update the `resume-schema` version inside the `package.json` to the latest version `0.1.3` 
* You also need to update this line in `serve.js` from 

https://github.com/mudassir0909/jsonresume-theme-elegant/blob/7d4262401132931f56919de94441fa864ba45333/serve.js#L11

to 

```
const resume = JSON.parse(fs.readFileSync('node_modules/resume-schema/sample.resume.json', 'utf8'));
```

With these changes in place, you should be able to see the projects when you do `grunt exec:run_server`. When I ran it seems to be missing the project name

Richard Hendriks 2020-09-14 19-16-21

I think with these changes in place we should be able to push forward to getting this PR merged very soon :)

File: index.js [hunk 1]
```diff
@@ -36,6 +36,30 @@ function render(resume) {
...
+
+        if (start_date.isValid()) {
+          project_info.startDate = utils.getFormattedDate(start_date);
+        }
+
+        if (end_date.isValid()) {
+          project_info.endDate = utils.getFormattedDate(end_date);
+        }
+
+        project_info.name = convertMarkdown(project_info.name);
```
Comment (mudassir0909) [inline index.js:56]: IMO we shouldn't allow markdown in the project's name. Visually, it should follow the styles specified by the theme

**Comment by vikytech** [comment]
@mudassir0909 Thanks for the reply, I will update the changes ASAP

File: index.js (hunk 1 above)
Comment (vikytech) [reply index.js:56]: Sure, Makes sense

**Comment by vikytech** [comment]
**Updates:**

1) Updated the resume-schema version to `0.1.3`
2) Fixed Project name rendring.
image

@mudassir0909 let me know if any changes to be done.

**Review by mudassir0909 — APPROVED** [review]

**Pull request merged**